### DISCUSSION TITLE
PBH 使用内存比qbittorent 还多

### DISCUSSION THREAD

**Discussion by lidongshengxdayu** [PBH-BTN/PeerBanHelper#704] [讨论]
在我环境里qbittorrent 有400+ 种子使用内存502M ，(虽然活跃的不多)

PBH 启动就600M+ ，等上一段时间能到900M+ `:-(`

我知道这是java 的问题，但是实在对小内存设备太不友好了
作者有没有计划用其他静态语言重写？或者已经有其他语言版本了我不知道？

**Comment by Ghost-chu** [comment]
1. 首先请确保你使用的 PBH 版本已更新到最新版本，我们在 7.1+ 做了大量修改，不太可能出现启动 600M+ 的情况
2. 你随时可以调整最大堆内存大小，在7.1+上，可稳定运行的最大堆内存大小最小是 386MB，默认为 512MB，但你可以尝试调整到 256MB
3. 没有其他语言重写的计划

**Reply by lidongshengxdayu** [reply]
检查了下之前用的6.4.8 , 更新7.1.2 后启动占用290M ，好了不少
